logstash-core/logstash-core.gemspec

```diff
@@ -44,6 +44,10 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency "rubyzip", "~> 1.1.7"
   gem.add_runtime_dependency "thread_safe", "~> 0.3.5" #(Apache 2.0 license)
 
+  # Required by the documentation tools
+  gem.add_runtime_dependency "asciidoctor", "~> 1.5.4"
+  gem.add_runtime_dependency "launchy", "~> 2.4.3"
```

Comment:
Hmm, I am a bit iffy about `launchy`. If it's not essential, and the license is not in our list, i would vote to remove it. You can always launch this stuff from your browser manually, right?